Tests/SwiftParserTest/translated/OperatorDeclTests.swift

```diff
@@ -167,7 +167,9 @@ final class OperatorDeclTests: XCTestCase {
       prefix operator %%+
       """,
       diagnostics: [
-        DiagnosticSpec(message: "unexpected code before operator declaration")
+        DiagnosticSpec(message: "operator must be declared as 'prefix', 'postfix', or 'infix'"),
+        // TODO: should be "unexpected code before operator declaration" here
+        DiagnosticSpec(message: "unexpected code in operator declaration"),
```

Comment:
I wonder whether there is a method can be used to mark a SyntaxCollection as a missing node when parsing? If not, does it make sense to create one? So that we can ignore the missing ``modifierList`` here in ``sourceAccurate`` mode and produce unexpected code **before** operator declaration rather than **in** declaration.

Reply:
Actually, the cause of the problem I mentioned above is that when we try to parse a modifier with a semicolon in front of, both the semicolon and the modifier will be parsed into an `UnexpectedNodeSyntax`. Although we could produce the correct diagnostic message, but the AST seems not correct to me, since the modifier is not unexpected. Maybe we could solve this problem in terms of recovery rather than the methods I mentioned above?